src/main/java/com/aircompanies/task/controller/AirCompanyController.java

```diff
@@ -1,12 +1,9 @@
 package com.aircompanies.task.controller;
```

Comment:
What task is it?

Reply:
![image](https://user-images.githubusercontent.com/78484371/146655836-be6770b3-ddc5-4ae5-a283-0cd23fe2a74d.png)